data/human/dirt belt jobs.txt

```diff
@@ -54,6 +54,7 @@ mission "To Remembrance Day celebration [0]"
 				random < 10
 				month == 2
 				day < 19
+		"reputation: Militia" >= 0
```

Comment:
Should we also check for the Governments of the planets departed from and landed upon? Those are either "Republic" or "Free Worlds" governed planets I guess.

Reply:
Militia is only friendly terms with both Republic and Free Worlds, I believe. However, I would expect that people in the dirt belt are more likely to trust the Militia's evaluation of someone than the Republic or (late enough in the plot) the Free World's evaluation: the Militia defends their planets, while the Republic is a far off government that doesn't do much for them but tax them, and the Free Worlds are an occupying force that may or may not have what's best for them in mind.